Gemfile

```
@@ -132,6 +132,6 @@ group :test, :development do
   # Load environment variables from .env
   gem 'dotenv-rails'
 
-  # Interactive debug tool
-  gem 'byebug'
+  # Call "buinding.pry" anywhere in the code to stop execution and get a debugger console
```

Should have spotted it earlier, but can always rely on @irisfaraway 

```suggestion
  # Call "binding.pry" anywhere in the code to stop execution and get a debugger console
```